src/etc/inc/openvpn.inc

```diff
@@ -719,6 +719,15 @@ function openvpn_reconfigure($mode, $settings) {
 				break;
 		}
 	}
+	if ($settings['dev_mode'] === 'tun' && isset($config['unbound']['enable']) && isset($config['unbound']['regovpnclients'])) {
```

Comment:
According to [man openvpn](https://openvpn.net/index.php/open-source/documentation/manuals/65-openvpn-20x-manpage.html) the address script parameter ...

> can be an IPv4 address such as "198.162.10.14", an IPv4 subnet such as "198.162.10.0/24", or an ethernet MAC address (when **--dev tap** is being used) such as "00:FF:01:02:03:04".

The script cannot handle MAC addresses (hence we cannot operate in `tap` device mode) and neither IPv4 subnets. I believe we never get those when the `mode` config is one of `server_tls` or `server_tls_user` (please correct me if I'm wrong).